tests/modules/did.test.ts

```diff
@@ -234,6 +233,75 @@ describe('DIDModule', () => {
 
             expect(didTx.code).toBe(0)
         }, defaultAsyncTxTimeout)
+
+        it('should create a new DID by controller', async () => {
+            const wallet = await DirectSecp256k1HdWallet.fromMnemonic(faucet.mnemonic, {prefix: faucet.prefix})
```

Comment:
It makes sense to extract DID creation in a helper method to reduce code duplication, in one way or another.
I think we can do this later, but let's add some `TODO` comment for this.

Reply:
I do not think that we should extract specifically DID creation because this test file actually verifies DID creation. But it makes sense to consider what else and how can be extracted to helpers over all `ledger-sdk` tests to make them shorter. At the same time, we should keep in mind that this project is a fork and significant refactoring will make it difficult to merge with upstream in future.